ui-tests/cypress/integration/administration/email-configuration-Sender.spec.js

```diff
@@ -0,0 +1,306 @@
+import HomePage       from '../../support/pageObjects/HomePage';
+import {htmlElements} from '../../support/pageObjects/WebElement';
+
+describe('Sender Management Functionalities', () =>{
+    let testCode = 'TestCode';
+      let testEmail = 'test@entando.com';
+      let defaultCode1 = 'CODE1';
+      let defaultEmail1 = 'EMAIL1@EMAIL.COM';
+      let defaultCode2 = 'CODE2';
+      let defaultEmail2 = 'EMAIL2@EMAIL.COM';
```

Comment:
Are you going to modify these values? If not, it would be better to define them as `const`